accessibility-checker-extension/src/ts/devtools/ReportRow.tsx

```diff
@@ -41,19 +41,25 @@ interface IReportRowProps {
     report: IReport,
     group: IReportRowGroup;
     selectItem: (item: IReportItem) => void,
-    getItem: (item: IReportItem) => void,
+    getItem: (item: IReportItem)=> void,
```

Comment:
Please maintain the spacing that linters use. There is no reason to remove the space before the `=>`

Reply:
added space